### DISCUSSION TITLE
Disappearing module with HTML video

### DISCUSSION THREAD

**Discussion by startslow** [microweber/microweber#932] [Q&A]
Hi, using Microweber 1.2.2, PHP 8 on Unix system.  Everything is working really well.  However I'm having a problem trying to use the stock 'Embed Code' module to add an HTML5 video.  The module adds, but, if the page or post is modified in the future using live edit, the module will disappear and needs to be re-added.  The page is saved properly after edit.

I've actually tried adding the HTML5 video through a variety of existing modules & methods and each time it disappears on editing the post in the future, adding the post to a new category will also cause the module to be removed.  The only way I've been able to get it to reliably save is by using a template, therefore having to edit a skin.php file which seems cumbersome for potentially many videos.  The way I'm doing it now is each video gets its own skin.  Is there a better or cleaner way to do this?  

Here is an example of the video code I need to somehow be saved, with the variables needed to edit mostly just the poster & two source lines:
<video controls="" height="" poster="...." preload="auto" width="">
<source src="..." type="video/mp4">
<source src="..." type="video/webm">
</video>

Thanks for any help or ideas which direction to go.

**Comment by startslow** [comment]
[html5example.txt](https://github.com/microweber/microweber/files/9366202/html5example.txt)

typed HTML5 code disappeared from this blog too so here it is in an attachment!

**Comment by peter-mw** [comment]
Hello,

Can you try to update your version to the latest ?

We tried to simulate this error but it does not happen here 

![image](https://user-images.githubusercontent.com/5698247/185380781-af5b4133-cc44-4749-bd48-c10a5e2eb378.png)

**Comment by startslow** [comment]
Hi,
  Thank for checking it out.  I updated to microweber 1.3.1 per instructions here: https://github.com/microweber/microweber/blob/master/UPGRADING.md

This also caused me to upgrade php80 to php81 due to an error from php composer, so I did that as well.

However the problem still exists.  The video adds, and saves, no problem.  It looks like I am using the same exact embed code module you are in the photo.  I was able to live edit the post once to change the content text and it remained, one time.  However, when I went to live edit it a second time, the module dropped.  Also when I edited any custom fields (I have a date field on the post) it also drops.  Switching the post to a new category also causes the module to reliably disappear.

Maybe i has something to do with mysql?  Happy to post whatever logs or try a next step.
